accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,550 @@
+{
+    public static bool IsSupported
+    {
+        get
+        {
+#if ANDROID || IOS || WINDOWS
+            return true;
+#else
+            return false;
+#endif
+        }
+    }
+
+    public static (double Latitude, double Longitude) GetCoordinates()
+    {
+#if ANDROID
+        return AndroidAPI();
+#elif IOS
+        return AppleAPI();
+#elif WINDOWS
+        return WindowsAPI();
+#else
+        throw new PlatformNotSupportedException();
+#endif
+    }
+}
+```
+
+### Vary API surface
+
+Ada is a developer on SkiaSharp, a cross-platform 2D graphics API for .NET
+platforms based on Google's Skia Graphics Library. The project is already using
+multi-targeting to provide different implementations for different platform. To
+make it easier to use, she's adding a new `SkiaSharpImage` type, which
+represents a bitmap and is constructed via OS-provided data types. Ada uses
+`#if` to expose different constructors on different platforms:
+
+```C#
+public static class SkiaSharpImage
+{
+#if ANDROID
+    public SkiaSharpImage(Android.Media.Image nativeImage) { /* ... */  }
+#endif
+
+#if IOS
+    public SkiaSharpImage(NSImage nativeImage) { /* ... */ }
+#endif
+
+#if WINDOWS
+    public SkiaSharpImage(Windows.Media.BitmapImage nativeImage) { /* ... */ }
+#endif
+}
+```
+
+## Requirements
+
+### Goals
+
+* Use naming that aligns with product strategy
+* Merge .NET Core and .NET Standard into a single concept
+* Developers should be able to understand compatibility relationships without
+  having to consult a mapping table.
+* Provide compatibility with existing concepts and NuGet packages
+* It would be great to get this into .NET 5 Preview 1
+
+### Non-Goals
+
+* Replace TFMs or expand runtime identifiers (RIDs)
+* Support multi-targeting between OS versions
+
+## Design
+
+| TFM             | Compatible With                                            | Comments                          |
+|-----------------|------------------------------------------------------------|-----------------------------------|
+| net5.0          | net1..4 (with NU1701 warning)                              | No WinForms or WPF                |
+|                 | netcoreapp1..3.1 (warning when WinForms/WPF is referenced) |                                   |
+|                 | netstandard1..2.1                                          |                                   |
+| net5.0-android  | xamarin.android                                            |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-ios      | xamarin.ios                                                |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-macos    | xamarin.macos                                              |                                   |
```

Comment:
Nit: The previous name is `xamarin.mac`, not `xamarin.macos`.

Reply:
Thanks, fixed.